tools/protoidl/parse.go

```diff
@@ -47,36 +51,26 @@ func GetMethods(content string) (methods []Method, err error) {
 	return methods, nil
 }
 
-func transferComment(lines []string) map[string]string {
-	commentMap := make(map[string]string)
+func transferComment(lines []string) (map[string]string, error) {
+	attributes := make(map[string]string)
 	for _, line := range lines {
 		index := strings.Index(line, "@Attribute")
 		if index == -1 {
 			continue
 		}
-		ss := line[index+10:]
-		key := []rune("")
-		value := []rune("")
-
-		split := false
-		for _, s := range ss {
-			switch s {
-			case ' ':
-				continue
-			case ':':
-				split = true
-				continue
-			default:
-				if !split {
-					key = append(key, s)
-				} else {
-					value = append(value, s)
-				}
-			}
+		ss := strings.SplitN(line[index+10:], ":", 2)
```

Comment:
So the attribute value may contain ':' ?

Reply:
yes, it's allowed.